examples/observability/eks-java-jmx/main.tf

```diff
@@ -3,19 +3,28 @@ provider "aws" {
 }
 
 provider "kubernetes" {
-  experiments {
-    manifest_resource = true
+  host                   = module.eks_blueprints.eks_cluster_endpoint
+  cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
+
+  exec {
+    api_version = "client.authentication.k8s.io/v1alpha1"
+    command     = "aws"
+    # This requires the awscli to be installed locally where Terraform is executed
+    args = ["eks", "get-token", "--cluster-name", module.eks_blueprints.eks_cluster_id]
   }
```

Comment:
What's the reason for this? Just curious..

Reply:
To follow the recommended practices by the provider https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs#exec-plugins

We also aligned the other examples on this approach as well recently

Reply:
Short lived tokens are by design. Not opposed to this change at this point but its not really fixing anything that's broken currently.